cvat/apps/dataset_manager/task.py

```diff
@@ -93,6 +94,12 @@ def add_prefetch_info(cls, queryset):
         ])
         label_qs = JobData.add_prefetch_info(label_qs)
 
+        task_data_queryset = models.Data.objects.all()
```

Comment:
I'm pretty sure you need only 1 Data object, as there is at most 1 related to a Task.